demo/molecule/pagination/demo/index.js

```diff
@@ -39,6 +39,14 @@ const Demo = () => {
     <div className="sui-StudioPreview">
       <div className="sui-StudioPreview-content sui-StudioDemo-preview">
         <h1>Pagination</h1>
+        <h2>Size</h2>
+        <div className={CLASS_DEMO_SECTION}>
+          <h3>Small</h3>
+          <MoleculePagination totalPages={25} page={17} size="small" />
+          <h3>Large</h3>
+          <MoleculePagination totalPages={25} page={17} />
+        </div>
+        {/* ------------------------------------------------------------------------------------------------------------- */}
```

Comment:
I think this is a kind of comment to separate the code? You could write something instead of only "----" so other developers will know why and what are you indicating

Reply:
This comment was already applied in different parts of the file, I guess they put it to differentiate the different parts of the demo.

I think it's good to remove them. Instead I leave a separation line in the code.